BlazorCore/DSD.MSS.Blazor.Components.Core/Models/RequestCommentInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DSD.MSS.Blazor.Components.Core.Models
+{
+    public class RequestCommentInfo
+    {
+        /// <summary>
+        /// Id for the comment
+        /// </summary>
+        public int Id { get; set; }
+        
+        /// <summary>
+        /// Work item ID attached this this comment
+        /// </summary>
+        public int WorkItemId { get; set; }
```

Comment:
This suggests to me that the component is tightly coupled with the WorkloadManagement API. This may be alright but just wanted to point it out.

Reply:
Removed this property